test/parallel_api/memory/specialized.algorithms/uninitialized_construct.pass.cpp

```diff
@@ -208,5 +207,10 @@ main()
     test_uninit_construct_by_type<std::int8_t>();
     test_uninit_construct_by_type<float64_t>();
 
+    // Regression tests to check compatibility of oneDPL implementation wit
```

Comment:
Typo (should be applied here and in other files):
```suggestion
    // Regression tests to check compatibility of oneDPL implementation with
```